src/main/java/de/qabel/desktop/storage/AbstractNavigation.java

```diff
@@ -66,18 +66,18 @@ public BoxNavigation navigate(BoxFolder target) throws QblStorageException {
 	public void commit() throws QblStorageException {
 		byte[] version = dm.getVersion();
 		dm.commit();
-		logger.info("Committing version "+ new String(Hex.encodeHex(dm.getVersion()))
-				+ " with device id " + new String(Hex.encodeHex(dm.deviceId)));
+		//logger.info("Committing version "+ new String(Hex.encodeHex(dm.getVersion()))
```

Comment:
I think the logging at `info` level is fine ... you could change it to `debug` level ... or delete it, but these comments are wasted with git ;)